Tem-Test/server.R

```diff
@@ -5,22 +5,21 @@ library(ggplot2)
 server<- function(input,output) {
   dataset<- reactive({
     selected_data<- data_list[as.integer(input$dataset),]
-    browser()
-    get(selected_data["Item"],pasteO("package:", selected_data["Package"]))
+    get(selected_data["Item"],paste0("package:", selected_data["Package"]))
   })
 
```

Comment:
Below in `DT::renderDT()` you have `x_var` and here you have `column_selection_x`. So for consistency I changed these to `x_var` and `y_var` (since those are easier to type than the longer names).